src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,47 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+}
```

Comment:
You can also get all Movie here
```suggestion
interface Props {
  movie: Movie;
}
```